app/schemas.py

```diff
@@ -16,6 +16,26 @@ class UserBase(BaseModel):
     token: str
 
 
+class LocationCreate(BaseModel):
+    model_config = ConfigDict(from_attributes=True, arbitrary_types_allowed=True)
+
+    osm_id: int = Field(gt=0)
+    osm_type: PriceLocationOSMType
+
+
+class LocationBase(LocationCreate):
+    id: int
+    osm_name: str | None
```

Comment:
It's not mandatory, but it's not clear what each field is about. It's possible to describe the field using pydantic `description` parameter in `Field`.

Reply:
i added this to #31